corehq/apps/linked_domain/updates.py

```diff
@@ -96,6 +110,50 @@ def update_custom_data_models(domain_link, limit_types=None):
         model.save()
 
 
+def update_fixtures(domain_link):
+    if domain_link.is_remote:
+        master_results = remote_fixtures(domain_link)
+    else:
+        master_results = local_fixtures(domain_link.master_domain)
+
+    skipped = []
+    linked_data_types_by_tag = {t.tag: t for t in get_fixture_data_types(domain_link.linked_domain)}
+    for master_data_type in master_results["data_types"]:
+        if not master_data_type.is_global:
+            skipped.append(master_data_type.tag)
+            continue
+
+        # Update data type
+        master_data_type = master_data_type.to_json()
+        master_data_type_id = master_data_type["_id"]
+        del master_data_type["_id"]
+        del master_data_type["_rev"]
+        linked_data_type = linked_data_types_by_tag.get(master_data_type["tag"], FixtureDataType()).to_json()
+        linked_data_type.update(master_data_type)
+        linked_data_type["domain"] = domain_link.linked_domain
+        linked_data_type = FixtureDataType.wrap(linked_data_type)
+        linked_data_type.save()
+        clear_fixture_quickcache(domain_link.linked_domain, [linked_data_type])
+
+        # Re-create relevant data items
+        delete_fixture_items_for_data_type(domain_link.linked_domain, linked_data_type._id)
```

Comment:
I don't know much about fixtures, but you're sure that they are never accessed by ID or something else that will get clobbered by this operation?

Reply:
Good question. The id isn't included in the restore, so apps can't refer to them by id, but I'll take a look at the HQ code that references fixtures.

Reply:
Just confirming that I looked around for fixture access by id and didn't find anything that wouldn't work with this code.